tests/src/test/scala/whisk/core/cli/test/WskConfigTests.scala

```diff
@@ -149,9 +149,7 @@ class WskConfigTests
                 // Send request to https://<apihost>/api/v1/namespaces, wsk client passes client certificate to nginx, nginx will
                 // verify it by client ca's openwhisk-client-ca-cert.pem
                 val stdout = wsk.cli(Seq("property", "set", "-i", "--apihost", wskprops.apihost, "--auth", wskprops.authKey,
-                    "--cert", wskprops.cert, "--key", wskprops.key, "--namespace", namespace), env = env).stdout
```

Comment:
Not so much what cert and key have to do with experimental, but what wskProps and WskPropsV2 have to do with the removal. The original wskprops was used by experimental (exclusively?), WskPropsV2 was used by non-experimental. @mdeuser asked me to clean up the 2 so now V2 is the only WskProps. Mark will have a better idea of exactly what this touches